v1/api/swagger/paths/projects@{projectId}.yaml

```
@@ -43,22 +45,25 @@ put:
   description: >-
     Updates a project with a matching ID. Only Project 'Name', 'Profile' and 'Limits' can be updated
     with this operation.
-    Note that Hoster or BMaaS Access Owner role is required for this operation.
+    Note that a Hoster, BMaaS Access Owner, BMaaS Tenant Owner or Service Platform Owner role 
+    is required for this operation.
```

Let's split this up into the separate environment
```suggestion
    Note that a Hoster or BMaaS Access Owner (under GreenLake Cloud Services), or
    BMaaS Tenant Owner or Service Platform Owner role (under GreenLake Platform) 
    is required for this operation.
```